common/src/Microsoft.Azure.IIoT.Agent.Framework/src/IJobOrchestratorEndpointSyncConfig.cs

```diff
@@ -4,15 +4,21 @@
 // ------------------------------------------------------------
 
 namespace Microsoft.Azure.IIoT.Agent.Framework.Jobs {
+    using System;
 
     /// <summary>
     /// Configuration of job orchestrator api endpoint
     /// </summary>
-    public interface IJobOrchestratorEndpoint {
+    public interface IJobOrchestratorEndpointSyncConfig {
```

Comment:
>Sync [](start = 45, length = 4)

nit: name might be better without 'Sync'

Reply:
Removed `Sync`.